source/reference/operator/query/mod.txt

```diff
@@ -117,3 +117,31 @@ operator with an array that contains four elements:
    	"$err" : "bad query: BadValue malformed mod, too many elements",
    	"code" : 16810
    }
+
+Modulo with Decimal Divisors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``modulo`` expression rounds down decimal input towards zero.
+
+The following examples demonstrates this behavior :
```

Comment:
[typo]

This should be "The following examples demonstrate..."

We also have an extra space between behavior and the colon.